master/reference/resources/hostendpoint.md

```diff
@@ -18,8 +18,10 @@ interfaces.
 Each host endpoint may include a set of labels and list of profiles that {{site.prodname}}
 will use to apply
 [policy]({{site.baseurl}}/{{page.version}}/reference/resources/networkpolicy)
-to the interface.  If no profiles or labels are applied, {{site.prodname}} will not apply
-any policy.
+to the interface.
+
+**Default behavior of external traffic to/from host**
```

Comment:
Markdown requires two returns for a new line. 
```suggestion
**Default behavior of external traffic to/from host**

```

Reply:
Fixed